### DISCUSSION TITLE
Does FreeRDP support to unlock the Windows Workstation and use existing session rather than disconnecting the existing one and creating a new one?

### DISCUSSION THREAD

**Discussion by ob-ab** [FreeRDP/FreeRDP#9811] [Q&A]
If it is supported then how can it be achieved? Or if this feature doesn't exist, can it be added?

**Comment by akallabeth** [comment]
that is a server side `RDP` feature, nothing to do with `FreeRDP`

**Reply by ob-ab** [reply]
Alright, thanks!
